server/helpers/middleware.js

```diff
@@ -43,6 +44,18 @@ export const authenticate = async (req, res, next) => {
       error,
     });
   }
+  return null;
+};
 
+export const isAdmin = async (req, res, next) => {
+  const { user } = req;
+  if (user.role === userRole.ADMIN) {
```

Comment:
Lets verify if user is valid here, so that we can get an error if used without authenticating (i.e. if user is null)
```suggestion
  if (user && user.role === userRole.ADMIN) {
```